packages/trader/src/Modules/Trading/Components/Elements/Turbos/payout-per-point.tsx

```diff
@@ -20,11 +21,12 @@ const PayoutPerPoint = observer(() => {
             i18n_default_text='<0>For {{title}}: </0>{{message}}'
             components={[<Text key={0} weight='bold' size='xxs' />]}
             values={{
-                title: contract_type === 'turboslong' ? localize('Long') : localize('Short'),
+                title: getContractSubtype(contract_key),
```

Comment:
if the method `getContractSubtype(contract_key)` is returning a string so we have to localize it for translation?